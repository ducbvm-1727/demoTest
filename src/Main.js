import React, { Component } from 'react'
import {
  View,
  Text,
  StyleSheet
} from 'react-native'

export default class Main extends Component {
  render() {
    return (
      <View>
        <Text style={styles.text}>{'My name is ' + `${this.props.name}`}</Text>
      </View>
    )
  }
}

const styles = StyleSheet.create({
  text: {
    fontSize: 28,
    alignItems: 'center',
    justifyContent: 'center'
  }
})
