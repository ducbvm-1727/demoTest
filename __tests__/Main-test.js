import 'react-native';
import React from 'react';
import Main from '../src/Main';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('Main snapshot', () => {
  const snap = renderer.create(
    <Main />
  ).toJSON();
  expect(snap).toMatchSnapshot();
});
