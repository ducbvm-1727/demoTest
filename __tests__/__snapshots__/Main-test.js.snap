// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main snapshot 1`] = `
<View>
  <Text
    style={
      Object {
        "alignItems": "center",
        "fontSize": 28,
        "justifyContent": "center",
      }
    }
  >
    My name is undefined
  </Text>
</View>
`;
